packages/data/abstract/userpoll/lib/src/poll.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+import 'package:poll_model/poll_model.dart';
+
+part 'poll.g.dart';
+
+@JsonSerializable()
+class UserPoll {
+  PollModel mood;
+  PollModel productivity;
+  PollModel relationships;
+  PollModel selfdevelopment;
```

Comment:
```suggestion
  PollModel selfDevelopment;
```

Reply:
In english it is one word self-development. It must be selfdevelopment, not selfDevelopment

Reply:
Isn't self-development =/= selfdevelopment? I don't really care though